compiler/rustc_trait_selection/src/traits/query/normalize.rs

```diff
@@ -353,18 +353,8 @@ impl<'cx, 'tcx> FallibleTypeFolder<'tcx> for QueryNormalizer<'cx, 'tcx> {
     ) -> Result<mir::ConstantKind<'tcx>, Self::Error> {
         Ok(match constant {
             mir::ConstantKind::Ty(c) => {
```

Comment:
now that's just the default impl, isn't it? :grin: 

don't think you have to overwrite this method at all now

Reply:
^^

Reply:
eh actually it isnt ^^: https://doc.rust-lang.org/nightly/nightly-rustc/src/rustc_middle/ty/fold.rs.html#193

Reply:
ah, then just `c.try_super_fold_with` i guess :grin: